docs/reference/mapping/runtime.asciidoc

```diff
@@ -0,0 +1,518 @@
+[[runtime]]
+== Runtime fields
+Typically, you index data into {es} to promote faster search. However, indexing
```

Comment:
This page is pretty long, would it make sense to split it into a couple of subpages/sections?

Reply:
I agree that it's a long page, but I don't want to chunk the content unnecessarily. I think there's a compromise of keeping Benefits, Use cases, and Compromises on the main page, and chunking the other sections to display in the right navigation.